packages/orbit-components/src/utils/Slide/README.md

```diff
@@ -16,11 +16,62 @@ import Slide from "@kiwicom/orbit-components/lib/utils/Slide";
-| transitionDuration | `string`         | `"fast"` | Determines the duration of the animation. Can be `"slow"`, `"normal"` or `"fast`.           |
-| children           | `React.Node`     |          | The expandable content that should be animated.                                             |
-| maxHeight          | `number \| null` |          | The maximum height the animation should take. Usually it is the height of the container.    |
-| expanded           | `boolean`        | `false`  | Determines if the content is expanded or not. When changed to `true`, the animation occurs. |
-| id                 | `string`         |          | Sets the id for the wrapper component responsible for the animation.                        |
-| ariaLabelledBy     | `string`         |          | Sets the ariaLabelledBy for the wrapper component responsible for the animation.            |
+| Name                 | Type             | Default  | Description                                                                                 |
+| :------------------- | :--------------- | :------- | :------------------------------------------------------------------------------------------ |
+| transitionDuration   | `string`         | `"fast"` | Determines the duration of the animation. Can be `"slow"`, `"normal"` or `"fast"`.          |
+| children             | `React.Node`     |          | The expandable content that should be animated.                                             |
+| maxHeight            | `number \| null` |          | The maximum height the animation should take. Usually it is the height of the container.    |
+| expanded             | `boolean`        | `false`  | Determines if the content is expanded or not. When changed to `true`, the animation occurs. |
+| id                   | `string`         |          | Sets the id for the wrapper component responsible for the animation.                        |
+| ariaLabelledBy       | `string`         |          | Sets the ariaLabelledBy for the wrapper component responsible for the animation.            |
+| stopClickPropagation | `boolean`        | `true`   | Prevents click events inside the component from propagating to parent elements.             |
+
+## Accessibility
+
+The `Slide` component implements several accessibility features to ensure proper interaction for all users:
+
+### ARIA Attributes
+
+- `aria-hidden`: Automatically set to `true` when the content is collapsed and `false` when expanded
+- `aria-labelledby`: When provided via the `ariaLabelledBy` prop, associates the collapsible content with its label element
+
+### Keyboard Navigation
+
+When using `Slide` for expandable components (like accordions or collapsible panels), ensure that:
+
+1. The control that toggles the `expanded` state is keyboard focusable
+2. The control has appropriate ARIA attributes (`aria-expanded`, `aria-controls`) pointing to the Slide component's `id`
+
```

Comment:
How about adding third point that if the control is the icon, it should be properly labelled?